components/brave_wallet_ui/components/desktop/popup-modals/bridge-to-aurora-modal/bridge-to-aurora-modal.tsx

```diff
@@ -23,25 +22,38 @@ interface Props {
   onOpenRainbowAppClick: () => void
 }
 
-const learnMoreLink = 'https://ethereum.org/en/bridges/#bridge-risk'
+const learnMoreLink = 'https://doc.aurora.dev/bridge/bridge-overview/'
+const learnMoreRiskMitigation = 'https://rainbowbridge.app/approvals'
 
 export const BridgeToAuroraModal = ({ onClose, onOpenRainbowAppClick }: Props) => {
   return (
     <PopupModal
       title=''
       onClose={onClose}
-      width={modalWidth}
     >
       <StyledWrapper>
         <Title>{getLocale('braveWalletAuroraModalTitle')}</Title>
-        <Description>{getLocale('braveWalletAuroraModalDescription')}</Description>
-        <LearnMoreLink
-          rel='noopener noreferrer'
-          target='_blank'
-          href={learnMoreLink}
-        >
-          {getLocale('braveWalletAuroraModalLearnMore')}
-        </LearnMoreLink>
+        <Description>
+          {getLocale('braveWalletAuroraModalDescription')}
+          <br/>
+          <LearnMoreLink
+            rel='noopener noreferrer'
+            target='_blank'
+            href={learnMoreLink}
+          >
+            {getLocale('braveWalletAuroraModalLearnMore')}
+          </LearnMoreLink>
+          &nbsp;
+          {getLocale('braveWalletAuroraModalLearnMoreOr')}
+          &nbsp;
+          <LearnMoreLink
+            rel='noopener noreferrer'
+            target='_blank'
+            href={learnMoreRiskMitigation}
+          >
+            {getLocale('braveWalletAuroraModalLearnMoreRiskMitigation')}
+          </LearnMoreLink>
```

Comment:
Good use case for the `splitStringForTag` method, otherwise this will not translate correctly.

```suggestion
const learnMoreTextParts = splitStringForTag(getLocale('stringWithEverythingTogether'))

const LearnMoreInfo = () => {
  return <>
          <LearnMoreLink
            rel='noopener noreferrer'
            target='_blank'
            href={learnMoreLink}
          >
            {learnMoreTextParts.beforeTag}
          </LearnMoreLink>
          &nbsp;
          {learnMoreTextParts.duringTag}
          &nbsp;
          <LearnMoreLink
            rel='noopener noreferrer'
            target='_blank'
            href={learnMoreRiskMitigation}
          >
            {learnMoreTextParts.afterTag}
          </LearnMoreLink>
  </>
}

  return (
    <PopupModal
      title=''
      onClose={onClose}
    >
      <StyledWrapper>
        <Title>{getLocale('braveWalletAuroraModalTitle')}</Title>
        <Description>
          {getLocale('braveWalletAuroraModalDescription')}
          <br/>
          <LearnMoreInfo />
```